api/envoy/config/trace/v2/trace.proto

```diff
@@ -197,6 +200,19 @@ message OpenCensusConfig {
   repeated TraceContext outgoing_trace_context = 9;
 }
 
+// Configuration for AWS X-Ray tracer.
```

Comment:
This should be tagged `[#not-implemented-hide:]` until the implementation is ready.